terraform/java/ecs/output.tf

```diff
@@ -0,0 +1,18 @@
+# ------------------------------------------------------------------------
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+# -------------------------------------------------------------------------
+
+output "sample_app_main_service_task_definition" {
+  value = aws_ecs_task_definition.main_service.arn
```

Comment:
I created this output because I thought it would be useful during trouble shooting, but wondering if it is the case.

If not, what should be put in the output?

Reply:
Similar as EKS test, I don't think we need a output, removed the `output.tf` file.